app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicsListAdapter.kt

```diff
@@ -48,20 +48,14 @@ class MusicListItemViewHolder(
 
     private fun favoriteStateHandler(obj: MusicRepoModel) {
         if (obj.isFavorite) mBinding.imgMusicFavorite.setImageResource(R.color.colorAccent)
```

Comment:
behtareh in check kardan yeh method joda neveshteh besh ta code tamiz tar bash.
barayeh onclick ham hamon call mishe.